wire.go

```diff
@@ -16,6 +16,8 @@ func init() {
 }
 
 func RegisterWire(cdc *wire.Codec) {
+	cdc.RegisterConcrete(Address{},
```

Comment:
should we be prefix encoding addresses?

Reply:
I don't know?

Reply:
No we shouldn't.  An address is just a []byte, happens to be hex encoded in JSON.  We don't want to register an Address, since it doesn't implement any interface (it's not 1 of anything).